core-tests/src/test/java/org/mule/runtime/core/DefaultMuleMessageTestCase.java

```diff
@@ -39,7 +40,7 @@
 {
 
     public static final String FOO_PROPERTY = "foo";
-    private Serializable testAttributes = new Apple();
+    private Attributes testAttributes = new Attributes() {};
```

Comment:
this will not be very supportable/debugable. When you see the toString(), or when you serialize, you will see an ugly name. It will also be hard to put a condition breakpoint... I would either make this an instantiable class or create a NullAttributes kind of thing